File: AR-VR/Vidushi7555_VidushiPandey_2024ec1126_2/GettingStarted/PrimeNumber.java
/******************************************************************************

                            Online Java Compiler.
                Code, Compile, Run and Debug java program online.
Write your code in this editor and press "Run" button to execute it.

*******************************************************************************/

import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
      int t=scn.nextInt();
      
      for(int i=1;i<=t;i++)
      {
          int n=scn.nextInt();
          
          int count=0;
          
          for(int j=2;j*j<=n;j++)
          {
              if(n%j==0)
              {
                  count++;
                  break;
              }
          }
          if(count==0)
          {
              System.out.println("prime");
          }
          else
          {
              System.out.println("not prime");
          }
          
      }
   }
  }

File: IOT/Vanshika7401_VanshikaSrivastava_2024it1076_2/GettingStarted/Prime._number.cpp
#include <iostream>
using namespace std;

bool isPrime(int n)
{
    
    if (n <= 1)
        return false;
  
    
    for (int i = 2; i < n; i++)
        if (n % i == 0)
            return false;
  
    return true;
}

int main(){
    int t;
    cin >> t;
   while(t--)
   {
       int n;
       cin>>n;
       isPrime(n)? cout << " is prime\n" : cout << " not prime\n";
   }
    //write your code here
    return 0;
}

  

  
